backend/src/test/java/com/soen341/instagram/dao/impl/CommentRepositoryTest.java

```diff
@@ -0,0 +1,58 @@
+package com.soen341.instagram.dao.impl;
+
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.model.Comment;
+import com.soen341.instagram.model.Picture;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
Nitpicky comment: add a line break before this import to separate the `.com` imports from the `.org` ones